confluent/docker_utils/cub.py

```diff
@@ -94,10 +95,13 @@ def __request(host, port, secure, ignore_cert, username='', password='', path=''
     return requests.get(url, verify = not ignore_cert, auth = auth)
 
 def log4j_config_file():
-    if (os.environ.get("COMPONENT")):
-        return "/etc/" + os.environ.get("COMPONENT") + "/log4j.properties"
-    else:
-        return "/etc/cp-base-new/log4j.properties"
+    def config_file = DEFAULT_LOG4J_FILE
+    def component_config = "/etc/" + os.environ.get("COMPONENT") + "/log4j.properties"
```

Comment:
you don't need `def` to define variables in Python:
```suggestion
    config_file = DEFAULT_LOG4J_FILE
    component_config = "/etc/" + os.environ.get("COMPONENT") + "/log4j.properties"
```

Reply:
i may or may not have been working in too much Groovy lately...